scripts/deploy-production.js

```diff
@@ -0,0 +1,126 @@
+const prompts = require("prompts");
+const { exec } = require("child_process");
+
+const localDeployScript =
+  "yarn build:prod && env-cmd -e production yarn deploy:do-not-use && env-cmd -e production ./scripts/email.sh";
```

Comment:
Given that `prod` is referenced in `package.json`, I'm guessing this is supposed to be:
```suggestion
  "yarn build:prod && env-cmd -e prod yarn deploy:do-not-use && env-cmd -e prod ./scripts/email.sh";
```

Parsley uses `production`/`devProduction` rather than `prod`/`devProd`, could be worth standardizing them

Reply:
Done will update the 1password file soon